src/d/actor/d_a_npc_photo.cpp

```diff
@@ -4,337 +4,2266 @@
 //
 
 #include "d/actor/d_a_npc_photo.h"
-#include "m_Do/m_Do_ext.h"
 #include "d/d_procname.h"
+#include "d/d_com_lib_game.h"
+#include "d/d_com_inf_game.h"
+#include "d/d_item.h"
+#include "d/d_a_obj.h"
+#include "d/d_path.h"
+#include "d/actor/d_a_player_main.h"
+#include "d/d_camera.h"
+#include "d/res/res_po.h"
+#include "d/d_snap.h"
+#include "d/d_kankyo_rain.h"
+#include "d/d_picture_box.h"
+#include "d/actor/d_a_tag_photo.h"
+
+#include "weak_bss_936_to_1036.h" // IWYU pragma: keep
+#include "weak_data_1811.h" // IWYU pragma: keep
+
+extern dCcD_SrcCyl dNpc_cyl_src;
+
+const char daNpcPhoto_c::m_arcname[] = "Auc";
+
+
+static const char* l_arcname_tbl[] = {
+    "Po"
+};
+
+static sPhotoAnmDat l_npc_anm_wait[] = {
+    0x00,
+    0x08,
+    0xFF,
+};
+
+static sPhotoAnmDat l_npc_anm_talk[] = {
+    0x01,
+    0x08,
+    0xFF,
+};
+
+static sPhotoAnmDat l_npc_anm_walk[] = {
+    0x03,
+    0x08,
+    0xFF,
+};
+
+static sPhotoAnmDat l_npc_anm_spit[] = {
+    0x04,
+    0x08,
+    0x01,
+
+    0x00,
+    0x08,
+    0xFF,
+};
+
+static sPhotoAnmDat l_npc_anm_talk2[] = {
+    0x02,
+    0x08,
+    0x03,
+
+    0x00,
+    0x08,
+    0xFF,
+};
+
+static sPhotoAnmDat l_npc_anm_talk4[] = {
+    0x02,
+    0x08,
+    0x01,
+    0x01,
+    0x08,
+    0xFF,
+};
```

Comment:
These are probably just single structs each, not arrays of structs.
```suggestion
static sPhotoAnmDat l_npc_anm_wait = {
    0x00,
    0x08,
    0xFF,
};

static sPhotoAnmDat l_npc_anm_talk = {
    0x01,
    0x08,
    0xFF,
};

static sPhotoAnmDat l_npc_anm_walk = {
    0x03,
    0x08,
    0xFF,
};

static sPhotoAnmDat l_npc_anm_spit = {
    0x04,
    0x08,
    0x01,

    0x00,
    0x08,
    0xFF,
};

static sPhotoAnmDat l_npc_anm_talk2 = {
    0x02,
    0x08,
    0x03,

    0x00,
    0x08,
    0xFF,
};

static sPhotoAnmDat l_npc_anm_talk4 = {
    0x02,
    0x08,
    0x01,
    0x01,
    0x08,
    0xFF,
};
```

Reply:
ive seen these in d_a_npc_people and d_a_npc_roten in the same way so are those also incorrect or are those correct and is the way it currently is in my file correct?

Reply:
Oh whoops, I missed that some of these are actually two elements long instead of one. In that case array is definitely right, but each element should have braces around it to distinguish them from one another, as right now it looks like they're all one element.
So e.g.
```cpp

static sPhotoAnmDat l_npc_anm_talk4 = {
    {
        0x02,
        0x08,
        0x01,
    },
    {
        0x01,
        0x08,
        0xFF,
    },
};
```

Reply:
i dont mean just that but even the on element long anmdat cases in other TU's are also defined as arrays 
such as 
``` static sPeopleAnmDat l_npc_anm_talk2_ug[] = {
    0x03,
    0x08,
    0x00,
};``` 
or 
``` static sRotenAnmDat l_npc_anm_wait[] = {
    0x00,
    0x0E,
    0xFF,
}; ```

Reply:
Yeah, the fact that some of them are more than 1 element long means they're all arrays, even the ones that are just 1 element long. I didn't see that when I posted the first review due to the missing brackets.

I just fixed npc_people and npc_roten to have the missing brackets now: https://github.com/zeldaret/tww/commit/3d5131231fa41587892780131fb04b9074881a1d